perf/perf_sdt_probe.py

```diff
@@ -51,7 +51,7 @@ def add_library(self):
         """
         self.libpthread = self.run_cmd_out("ldconfig -p")
         for line in str(self.libpthread).splitlines():
-            if re.search('libpthread', line, re.IGNORECASE):
+            if re.search('libpthread.so', line, re.IGNORECASE):
```

Comment:
is this work in RHEL8.* and RHEL9.0 as well, if not please restrict this to RHEL9.1

Reply:
Its working on RHEL8.* RHEL9.0 and SLES also

on RHEL8.*
avocado run --test-runner runner perf_sdt_probe.py 
JOB ID     : 3cab9b8898d5fc78cba42cd015766e75b34611a1
JOB LOG    : /home/disha/avocado-fvt-wrapper/results/job-2022-06-20T11.26-3cab9b8/job.log
 (1/1) perf_sdt_probe.py:PerfSDT.test: PASS (6.44 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/disha/avocado-fvt-wrapper/results/job-2022-06-20T11.26-3cab9b8/results.html
JOB TIME   : 7.12 s

on RHEL 9.0
avocado run --test-runner runner perf_sdt_probe.py
JOB ID     : a1528954f077cef8b4deda4aebd688df9d6c33b5
JOB LOG    : /home/disha/avocado-fvt-wrapper/results/job-2022-06-21T15.57-a152895/job.log
 (1/1) perf_sdt_probe.py:PerfSDT.test: PASS (5.23 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/disha/avocado-fvt-wrapper/results/job-2022-06-21T15.57-a152895/results.html
JOB TIME   : 13.91 s

on SLES15
avocado run --test-runner runner perf_sdt_probe.py 
JOB ID     : daae2da8f2388d6642ee4d4c4f77a68549eb4239
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2022-06-21T10.40-daae2da/job.log
 (1/1) perf_sdt_probe.py:PerfSDT.test: PASS (8.03 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2022-06-21T10.40-daae2da/results.html
JOB TIME   : 18.71 s